tools/api-generator/convert.go

```diff
@@ -273,6 +267,47 @@ func extractParametersNameDescription(parameterRef *openapi3.ParameterRef) (stri
 	return parameterName, parameterDescription
 }
 
+type parameterExtensions struct {
+	aliases []string
+	short   string
+}
+
+func extractParameterExtensions(parameterRef *openapi3.ParameterRef) parameterExtensions {
+	ext := parameterExtensions{
+		aliases: []string{},
+		short:   "",
+	}
+
+	getExtensionValues := func(extensionsMap map[string]any) {
```

Comment:
any particular reason for an anonymous function? doesn't seem to be used as a callback or similar pattern of passing as an arg

Reply:
that's probably a tell we may need better code organization, in lining the function because trying to name it causes confusing with other functions sound like a good time to think if this package needs some cleaning

Reply:
I've reorganized the code to get rid of the anonymous function